src/test/resources/azure/main.tf

```diff
@@ -0,0 +1,70 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_mssql_server" "sql_server" {
+  name                = var.server_name
+  resource_group_name = var.resource_group_name
+  location            = var.location
+  version             = "12.0"
+  administrator_login = var.admin_login_name
+  #administrator_login_password = random_string.password.result
```

Comment:
@ap-liquibase I think we'd prefer to have the random pw here

Reply:
ack. yes. I will correct